src/lib/utils/template_generator.js

```diff
@@ -128,6 +128,18 @@ class TemplateGenerator {
   getExternalProject(uri) {
     let url, folder, hgi;
     let fallback, url_fallback, folder_fallback, hgi_fallback, embarkVersion;
+    const replacements = {
+      'git@github.com/': 'git@github.com:',
+      'git@bitbucket.org/': 'git@bitbucket.org:',
+      'git@gitlab.com/': 'git@gitlab.com:',
+      'github.com/': '',
+      'bitbucket.org/': '',
+      'gitlab.com/': ''
```

Comment:
> In that case, the hosted-git-info library picks it up as a github shortcut and is then able to parse it properly.

I think it'd be useful to have this (or a shorter version) comment appended to one of these last two replacements.

Happy to trade it against the

> // reformat uri before parsing with hosted-git-info. Allows for further syntax support.

Comment a few lines below :)